### PR TITLE
Suggest missing questions for js

### PR DESCRIPTION
**Pull request by lrecknagel**

### CODE REVIEW

File: gamefiles/ae.json [hunk 1]
```diff
@@ -195,15 +195,15 @@
                   },
                   {
                       "id": 24,
-                      "answer": "AnswerString5-4",
-                      "question": "QuestionString",
+                      "answer": "Proxies.",
+                      "question": "Who is JavaScripts magic guard?",
```
Comment (MarlonSchultz) [inline gamefiles/ae.json:199]: Should this be "What is?"

File: gamefiles/ae.json [hunk 2]
```diff
@@ -195,15 +195,15 @@
...
                       "points": 400,
                       "type": "question"
                   },
                   {
                       "id": 25,
-                      "answer": "AnswerString5-5",
-                      "question": "QuestionString",
+                      "answer": "The FinalizationRegistry",
+                      "question": "Which is a somehow unreliable, likely gargabe, registry?",
```
Comment (MarlonSchultz) [inline gamefiles/ae.json:206]: What is? Or is this meant as a joke that I am not getting?

Comment (lrecknagel) [reply gamefiles/ae.json:206]: will fix it.

File: gamefiles/ae.json (hunk 1 above)
Comment (lrecknagel) [reply gamefiles/ae.json:199]: will fix it.

**Pull request merged**